impeller/entity/entity_pass.cc

```diff
@@ -161,29 +161,31 @@ std::optional<Rect> EntityPass::GetElementsCoverage(
       if (image_filter) {
         Entity subpass_entity;
         subpass_entity.SetTransformation(subpass.xformation_);
-        coverage = image_filter->GetCoverage(subpass_entity);
+        element_coverage = image_filter->GetCoverage(subpass_entity);
       } else {
-        coverage = unfiltered_coverage;
+        element_coverage = unfiltered_coverage;
       }
 
-      if (coverage.has_value() && coverage_limit.has_value() &&
+      if (element_coverage.has_value() && coverage_limit.has_value() &&
           (!image_filter || image_filter->IsTranslationOnly())) {
-        coverage = coverage->Intersection(coverage_limit.value());
+        element_coverage =
+            element_coverage->Intersection(coverage_limit.value());
       }
     } else {
       FML_UNREACHABLE();
     }
 
-    if (!result.has_value() && coverage.has_value()) {
-      result = coverage;
+    if (!accumulated_coverage.has_value() && element_coverage.has_value()) {
+      accumulated_coverage = element_coverage;
       continue;
     }
-    if (!coverage.has_value()) {
+    if (!element_coverage.has_value()) {
       continue;
     }
-    result = result->Union(coverage.value());
+    accumulated_coverage =
+        accumulated_coverage->Union(element_coverage.value());
```

Comment:
All of these lines (178-187) => `accumulated_coverage = Union(accumulated_coverage, element_coverage)`

Reply:
Done.